docs/store-devtools/README.md

```diff
@@ -16,7 +16,7 @@ Install @ngrx/store-devtools from npm:
 
 ### Instrumentation with the Chrome / Firefox Extension
 
-1.  Download the [Redux Devtools Extension](http://zalmoxisus.github.io/redux-devtools-extension/)
+1.  Download the [Redux Devtools Extension](https://github.com/zalmoxisus/redux-devtools-extension/)
```

Comment:
I think the link here should be, https://extension.remotedev.io/ ?

Reply:
I see, I just mentioned it because this is the page where we're currently being redirect to.